robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenuItem.java

```diff
@@ -8,7 +9,6 @@
 import android.view.SubMenu;
 import android.view.View;
 
-import org.robolectric.RuntimeEnvironment;
 import org.robolectric.shadows.ShadowApplication;
```

Comment:
Would you mind switching ShadowApplication.getInstance().startActivity() to context.startActivity() then we can remove the dependency on ShadowApplication / shadows from this class.